src/murfey/server/__init__.py

```diff
@@ -2964,6 +2971,29 @@ def feedback_callback(header: dict, message: dict) -> None:
             if _transport_object:
                 _transport_object.transport.ack(header)
             return None
+        elif message["register"] in (
+            "register_lif_preprocessing_result",
+            "register_tiff_preprocessing_result",
+        ):
+            murfey_workflows = list(
+                entry_points().select(
+                    group="murfey.workflows.clem", name=message["register"]
+                )
+            )
+            # Run the workflow if a match is found
+            if len(murfey_workflows) > 0:
+                workflow: EntryPoint = murfey_workflows[0]
```

Comment:
This should probably loop over all found entry points rather than just running the first one

Reply:
Sorry, I misread what it was doing slightly and thought it was checking all the "murfey.workflows.clem" entry points. You're right it will only find one entry so fine to just run that one.
Up to you whether you want to search through the other clem ones too